src/main/Config.cpp

```diff
@@ -1558,11 +1558,6 @@ Config::getExpectedLedgerCloseTime() const
 void
 Config::setInMemoryMode()
 {
-    // For the time being, --in-memory mode on the command line does not enable
-    // MODE_USES_IN_MEMORY_LEDGER; rather it uses in-memory sqlite. This change
-    // happened when mitigating the protocol 16 issue, funneling all database
-    // access through the sqlite layer; it may be reverted once a different
-    // workaround for that issue is in place.
```

Comment:
Nitpick: might considering squashing this into the previous commit. I added a comment saying this was stale, only to see it fixed in the next commit.

Reply:
done.